src/vis/vis.js

```diff
@@ -317,15 +317,9 @@ var Vis = View.extend({
       provider: data.map_provider
     };
 
-    // if the boundaries are defined, we add them to the map
-    if (data.bounding_box_sw && data.bounding_box_ne) {
```

Comment:
There is no more uses of those variables in the current code and viz.json doesn't provide them, only bounds (and center + zoom) related to map centering.